code/game/machinery/computer/abnormality_archive.dm

```diff
@@ -0,0 +1,89 @@
+/obj/machinery/computer/abnormality_archive
+	name = "abnormality archive"
+	desc = "A secure terminal connected to the Lobotomy Corp archive."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "computer"
+	var/printfile
+	var/printing_status
+	var/notehtml
+	var/note
+	var/list/note_list
```

Comment:
The pedantic in me requires that I ask that lists and normal vars be put after one another for purely aesthetic reasons